example/spec/parallel/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -389,7 +403,7 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
     });
   });
 
-  describe('Workflow fails because SyncGranule fails', () => {
+  xdescribe('Workflow fails because SyncGranule fails', () => {
```

Comment:
Why did you disable this?

Reply:
Accidentally committed. While I was testing locally I disabled other parts of the test that weren't relevant for proving that my changes were successful

Reply:
f7ba969683bf1136e46716a06a4c7f04011ec576